testapp/src/main/java/com/rakuten/tech/mobile/testapp/analytics/rat_wrapper/RATButton.kt

```diff
@@ -14,6 +14,7 @@ import com.rakuten.tech.mobile.testapp.ui.settings.AppSettings
 class RATButton : AppCompatButton, IRatComponent {
```

Comment:
For buttons, we also need to send an `appear` event when the button first loads on the screen.